Makefile

```diff
@@ -62,7 +62,7 @@ CLEANLIBS = $(ARLIB)
 CLEANOBJS = $(OBJ_FILES)
 CLEANFILES = $(PGDIRBZ2)
 
-AR = ar rs
+AR := $(AR) rs
```

Comment:
Do we need to treat this like we treat CC below, which is to default it to `ar` explicitly?

i.e. make it

```
AR ?= ar
AR := $(AR) rs
```

Reply:
I suppose it doesn't hurt! I left the first part out based on these `Makefile` docs:
https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html

But not 100% if those defaults work in all environments. Will update with your suggestion.